app/lib/admin/metrics/dimension/space_usage_dimension.rb

```diff
@@ -27,11 +27,18 @@ def postgresql_size
   end
 
   def redis_size
+    human_key = if redis_info.key?('valkey_version')
+                  'Valkey'
+                elsif redis_info.key?('dragonfly_version')
+                  'Dragonfly'
+                else
+                  'Redis'
+                end
```

Comment:
Since this is repeated in both classes, what would you think of extracting this? You could create a mixin similar to the existing `query_helper.rb` in the same directory and add a method for this and also move the `#redis_info` method into there.

Both the `SoftwareVersionsDimension` and the `SpaceUsageDimension` class would then only need to include that module.